public/assets/css/patWaitingRoom.css

```diff
@@ -84,10 +84,13 @@ button {
   margin-top:125px;
 /*   border: 1px solid black; */
 }
+
+/* This is an empty rule set. Is there anything that is supposed to go inside this ID? */
```

Comment:
If the app works with this being empty, you can probably remove it :)

Reply:
@gideoncodenext Can you decide what you'll do about this? Delete it (verify that nothing breaks) or keep it (and merge this PR).

Thanks, Joe